src/v/config/configuration.cc

```diff
@@ -259,6 +259,21 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       64,
       {.min = 1, .max = 16384})
+  , raft_replica_max_not_flushed_bytes(
+      *this,
+      "raft_replica_max_not_flushed_bytes",
+      "Max not flushed bytes per partition. If configured threshold is reached "
+      "log will automatically be flushed even tho it wasn't explicitly "
+      "requested",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      256_KiB)
+  , raft_flush_timer_interval_ms(
+      *this,
+      "raft_flush_timer_interval_ms",
+      "Flush manager interval, Raft partitions will not be check if they "
```

Comment:
It is a bit unclear from the description and the property name - is this the interval of checks against `raft_replica_max_not_flushed_bytes` or will the log be flushed even if the number of flushed bytes is smaller?